assets/js/modules/optimize/util/validation.js

```diff
@@ -37,5 +37,18 @@ export function isValidOptimizeID( optimizeID ) {
  * @return {boolean} True if the given ampExperimentJSON is valid, false otherwise.
  */
 export function isValidAMPExperimentJSON( ampExperimentJSON ) {
-	return typeof ampExperimentJSON === 'string';
+	if ( undefined === ampExperimentJSON || '' === ampExperimentJSON || ! ampExperimentJSON ) {
```

Comment:
The last check here `! ampExperimentJSON` covers the first two, but would also allow for another falsy value like `0` to be considered valid which should not. This probably wouldn't be possible from user input since that should all come as a string from the field but this utility shouldn't assume that.

Reply:
Done